src/pages/components/notifications.mdx

```diff
@@ -53,6 +53,15 @@ import TypeCode from '!!raw-loader!../../examples/notifications/notifications/No
   <Type />
 </CodeExample>
 
+### Toasts
+
+import Toast from '../../examples/notifications/toasts/Toast.tsx';
+import ToastCode from '!!raw-loader!../../examples/notifications/toasts/Toast.tsx';
+
+<CodeExample code={ToastCode}>
+  <Toast />
+</CodeExample>
+
```

Comment:
@jzempel is there any documentation/copy created for this area? I'm not sure if this will overlap with the related pattern work.

Reply:
Yeah, I'm going to have to work with @allisonacs and @nyckie to determine how we want to talk about this here. In the mean time, can the associated component prop sheet(s) be added to the API section below?

Reply:
Updated to now include the prop-sheet for `ToastProvider`